subsys/net/l2/wifi/wifi_nm.ld

```diff
@@ -4,4 +4,6 @@
  * SPDX-License-Identifier: Apache-2.0
```

Comment:
Please change the commit subject to
`net: wifi: Add iterable_sections.h header file`
or something like that. The patch changes files in net, and not in iterable_sections.